File: exercise/php/day13/src/SantaMarket/ProductQuantity.php
<?php

namespace SantaMarket;

class ProductQuantity
{
    private Product $product;
    private float $quantity;

    public function __construct(Product $product, float $quantity)
    {
        $this->product = $product;
        $this->quantity = $quantity;
    }

    public function getProduct(): Product
    {
        return $this->product;
    }

    public function getQuantity(): float
    {
        return $this->quantity;
    }
}

File: exercise/C#/day13/SantaMarket/Model/ProductQuantity.cs
namespace SantaMarket.Model
{
    public class ProductQuantity(Product product, double quantity)
    {
        public Product Product { get; } = product;
        public double Quantity { get; } = quantity;

        public override string ToString() => $"ProductQuantity{{product={Product}, quantity={Quantity}}}";
    }
}